spec/models/profile.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require './spec/models/user'
+
+class Profile < PorkyLib::ApplicationRecord
```

Comment:
these test models are very specific to PorkyLib use in RR. Instead I would create classes in the spec file(s) and test different ways of configuring them

Reply:
I added some test cases (haven't pushed it yet), but I'm not sure how to define a relationship in classes defined this way in a spec file.

```
let(:class_name) {
  Class.new(PorkyLib::ApplicationRecord) do
  end
}
```

Maybe I'm not doing it the way you had in mind, please let me know, but I feel like I'm not able to do it without a schema file / models

Reply:
I don't see this file anymore; unsure if still relevant?